docs/building.md

```diff
@@ -0,0 +1,125 @@
+# Building a SeedSigner SD card image
+Assemble the SeedSigner OS along with the SeedSigner application code into an image file that can be flashed to an SD card and run in your SeedSigner.
+<br/>
+<br/>
+<br/>
+## 🔥🔥🔥🛠 Quickstart: SeedSigner Reproducible Build! 🛠🔥🔥🔥
+
+### Install Dependencies
+* Docker (choose one):
+    * Desktop users: [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+    * Or Linux command line: [Docker Engine](https://docs.docker.com/engine/install/#server)
+
+Windows users may also need to [install `git`](https://git-scm.com/download/win)
```

Comment:
The Docker docs already suggest running their intro container after the install steps. There's an endless level of hell in trying to anticipate/debug every possible problem that might occur along the way so that's definitely out of scope for this "Quickstart" section.